packages/marathon/build

```diff
@@ -24,17 +35,30 @@ EnvironmentFile=/opt/mesosphere/environment
 EnvironmentFile=/opt/mesosphere/etc/marathon
 EnvironmentFile=-/opt/mesosphere/etc/marathon-extras
 EnvironmentFile=-/var/lib/dcos/marathon/environment.ip.marathon
+Environment=JAVA_HOME=${JAVA_HOME}
 ExecStartPre=/bin/ping -c1 leader.mesos
 ExecStartPre=/opt/mesosphere/bin/bootstrap dcos-marathon
 ExecStartPre=/bin/bash -c 'echo "HOST_IP=\$(\$MESOS_IP_DISCOVERY_COMMAND)" > /var/lib/dcos/marathon/environment.ip.marathon'
 ExecStartPre=/bin/bash -c 'echo "MARATHON_HOSTNAME=\$(\$MESOS_IP_DISCOVERY_COMMAND)" >> /var/lib/dcos/marathon/environment.ip.marathon'
 ExecStartPre=/bin/bash -c 'echo "LIBPROCESS_IP=\$(\$MESOS_IP_DISCOVERY_COMMAND)" >> /var/lib/dcos/marathon/environment.ip.marathon'
-ExecStart=/opt/mesosphere/bin/java \\
-    -Xmx2G \\
-    -jar "$PKG_PATH/usr/marathon.jar" \\
-    --zk "\$MARATHON_ZK" \\
+ExecStart=$PKG_PATH/marathon/bin/marathon \\
+    -J-XX:+UseConcMarkSweepGC \\
+    -J-XX:+CMSClassUnloadingEnabled \\
+    -J-server \\
+    -J-XX:+CMSParallelRemarkEnabled \\
+    -J-XX:+UseCMSInitiatingOccupancyOnly \\
+    -J-XX:+ScavengeBeforeFullGC \\
+    -J-XX:+CMSScavengeBeforeRemark \\
+    -J-XX:+ScavengeBeforeFullGC \\
+    -J-XX:+CMSScavengeBeforeRemark \\
+    -J-XX:SoftRefLRUPolicyMSPerMB=50 \\
+    -J-XX:CMSInitiatingOccupancyFraction=70 \\
+    -J-XX:-OmitStackTraceInFastThrow \\
+    -J-XX:+AggressiveOpts \\
+    -J-verbose:gc \\
+    -J-XX:+PrintGCDetails \\
+    -J-XX:+PrintGCTimeStamps \\
```

Comment:
I'm curious how these were determined and how they ended up here.   I question the move to CMS (it has a negative impact to minor gc (which may be ok))
however verbose:gc and PrintGCDetails is something you usually reserve for debugging not for everyday production.  
and `AggressiveOpts` is really questionable